### ISSUE TITLE
Support SourceTree history

### ISSUE THREAD

**Issue by fzumstein** [ZoomerAnalytics/git-xltrail#33]
SourceTree since v3.0.12 supports diffing of uncommitted changes. We should also make this work for past commits. See also: https://community.atlassian.com/t5/Sourcetree-questions/SourceTree-VBA-Macros-via-xltrail/qaq-p/818400

**Comment by fzumstein** [comment]
apparently, nothing to do for us (as per above thread):

```
nothing you need to do on your side -- looks like 'diff-tree' doesn't have the default diff settings of '--ext-diff' like 'log' and 'diff' does for some reason. I'll add it to our backlog and hopefully it will be included in the next release.
```

**Comment by fzumstein** [comment]
This is now fixed in SourceTree 3.1.3
